gp2gp-translator/src/main/java/uk/nhs/adaptors/pss/translator/service/ConfidentialityService.java

```diff
@@ -20,7 +21,7 @@ public class ConfidentialityService {
 
     @SafeVarargs
     public final Meta createMetaAndAddSecurityIfConfidentialityCodesPresent(String metaProfile, Optional<CV>... cvs) {
-        return Arrays.stream(cvs).flatMap(Optional::stream).anyMatch(this::isNopat)
+        return Arrays.stream(cvs).filter(Objects::nonNull).flatMap(Optional::stream).anyMatch(this::isNopat)
```

Comment:
@ole4ryb Did you notice this `pitest` message? Seems this change is unnecessary, at least according to our tests.

Reply:
ok, I will remove it.